src/styles/theme.ts

```diff
@@ -337,6 +338,21 @@ const initTheme = (darkMode: boolean) => {
           },
         },
       },
+      MuiFilledInput: {
+        styleOverrides: {
+          root: ({ theme }) => ({
+            borderRadius: 10,
+            backgroundColor: theme.palette.background.main,
+            border: '1px solid transparent',
+            transition: 'border-color 0.2s',
+
+            '&:hover, &:focus, &.Mui-focused': {
+              backgroundColor: theme.palette.background.main,
+              borderColor: theme.palette.primary.main,
+            },
+          }),
+        },
+      },
```

Comment:
Does this override only apply to the search field? If so, I'd avoid setting it as a global override.

Reply:
At the moment, yes, the search field is the only filled input we use. But i'd imagine if we use a filled input in any other place it'd follow the same style. So I was careful by setting any properties that can be harmful globally (width, z-index, etc.)

Reply:
Sounds good. I'd change `borderRadius` to `"4px"` so that it matches that of the outlined input default (maybe we should also set that in the `MuiOutlinedInput` override in case they change it in the future).

Reply:
Turns out `10px` was an old design, not it's back to `4px`